### PR TITLE
repair button script

### PR DESCRIPTION
**Pull request by qwuae1**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by zhoxing-ms** [comment]
Could you add some test cases for this new feature?

**Comment by yonzhan** [comment]
Please fix conflicting files

File: src/vm-repair/azext_vm_repair/tests/latest/test_repair_commands.py [hunk 1]
```diff
@@ -1,850 +0,0 @@
...
-        # Something wrong with vm create command if it fails here
-        assert len(vms) == 1
-
-        # Test create
-        result = self.cmd('vm repair create -g {rg} -n {vm} --repair-username azureadmin --repair-password !Passw0rd2018 -o json --yes').get_output_in_json()
-        assert result['status'] == STATUS_SUCCESS, result['error_message']
-
-        # Check repair VM
-        repair_vms = self.cmd('vm list -g {} -o json'.format(result['repair_resource_group'])).get_output_in_json()
-        assert len(repair_vms) == 1
```
Comment (zhoxing-ms) [inline src/vm-repair/azext_vm_repair/tests/latest/test_repair_commands.py:36]: Why did you remove these tests?

**Review by zhoxing-ms — APPROVED** [review]
